File: TravelBois2/WebApp/Models/Pozivnica.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace WebApp.Models
{
    public class Pozivnica
    {
        [Key]
        [Required]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        [Required]
        public string Ime { get; set; }
        [Required]
        public string Prezime { get; set; }
        [Required]
        public string BrojPasosa { get; set; }
        [Required]
        public bool Rezervisano { get; set; }
        [Required]
        public int IdLeta { get; set; }
        [Required]
        public string IdSedista { get; set; }
        [Required]
        public int CenaSedista { get; set; }
        [Required]
        public string PozvaoUsername { get; set; }

    }
}

File: TravelBois2/WebApp/ClientApp/src/app/entities/objects/pozivnica.ts
export class Pozivnica {
  idLeta: number;
  ime: string;
  prezime: string;
  brojPasosa: string;
  rezervisano: boolean;
  cenaSedista: number;
  idSedista: string;
  pozvaoUsername: string;
  id: number;

  constructor(idLeta: number, idSedista: string, ime: string, prezime: string, brojPasosa: string, rezervisano: boolean, cenaSedista: number, pozvaoUsername: string) {
    this.idLeta = idLeta;
    this.ime = ime;
    this.prezime = prezime;
    this.brojPasosa = brojPasosa;
    this.rezervisano = rezervisano;
    this.cenaSedista = cenaSedista;
    this.idSedista = idSedista;
    this.pozvaoUsername = pozvaoUsername;
  }
}
